lib/beaker-rspec/helpers/serverspec.rb

```diff
@@ -72,7 +72,6 @@ def os
   def detect_os
     return Specinfra.configuration.os if Specinfra.configuration.os
 
-    backend = Specinfra.backend
```

Comment:
I think https://github.com/voxpupuli/beaker-rspec/pull/145 was actually intended when this was written.